docs/02_database/05_database-interface/01_entitydb.md

```diff
@@ -119,6 +167,22 @@ final var trade = db.get(trade.byTypeId())
         .blockingGet();
 ```
 </TabItem>
+<TabItem value="java-sync">
+
+```java
+// we can look up trades by passing in a unique index class:
+final var trade = db.get(Trade.byId("TRADE_1"))
+
+// a trade object with the primary key set 
+final var trade = db.get(trade)
+
+// a trade object and a reference to unique index
+final var trade = db.get(trade,Trade.ByTypeId.Companion)
```

Comment:
do we need to add more info about the blocking API itself? It is important to explain some details like the fact that we can return lists or streams for our bulk/range operations.